gpt_2_simple/gpt_2.py

```diff
@@ -721,6 +737,9 @@ def cmd():
     parser.add_argument(
         '--sample_delim',  help="[generate] Delimiter between each generated sample.",
         nargs='?', default='=' * 20 + '\n', type=str)
+    parser.add_argument(
+        '--multi_gpu',  help="[generate/finetune] Attempt to allocate multiple GPUs for running.",
+        nargs='?', default='=' * 20 + '\n', type=bool)
```

Comment:
I think, `default` & `type` need to be:
```
default=True, type=lambda x: (str(x).lower() == 'true')
```

because now every time we get multi_gpu = True